### DISCUSSION TITLE
Next Release version?

### DISCUSSION THREAD

**Discussion by Victor-x1** [microsoft/MixedRealityToolkit-Unity#11335] [Q&A] [answered]
Is the next release going to be pre13 or it going to be GA 1.0? Pre.12 was released about 8 weeks ago. Thank you for answering this question in advance!

**Comment by Zee2** [comment, marked as answer]
Hey Victor. Sorry we're behind on preview releases. A lot of folks on the team have been out of office for the past few weeks. Next release is very soon (pre.13). GA targeting early Feb.

**Reply by Zee2** [reply]
@Victor-x1, here it is! https://github.com/microsoft/MixedRealityToolkit-Unity/discussions/11351
